src/components/IncidentDetails/DefinitionOfDone/index.tsx

```diff
@@ -0,0 +1,347 @@
+      <Modal
+        title="Definition of Done"
+        onClose={() => {
+          setDODModalOpen(false);
+        }}
+        open={dodModalOpen}
+        bodyClass=""
+      >
+        <div
+          style={{ maxHeight: "calc(100vh - 6rem)" }}
+          className="py-4 px-8 overflow-y-auto overflow-x-hidden divide-y divide-gray-200"
+        >
+          {dodEvidences.map((evidence, index) => {
+            return (
+              <div key={index} className="py-6">
+                <EvidenceItem evidence={evidence} />
+              </div>
+            );
+          })}
+        </div>
+      </Modal>
+      <Modal
+        title="Add to Definition of done"
+        onClose={() => {
+          setAddToDODModalOpen(false);
+          setSelectedEvidences([]);
+        }}
+        open={addToDODModalOpen}
+        bodyClass=""
+      >
+        <div
+          style={{ maxHeight: "calc(100vh - 6rem)" }}
+          className="overflow-y-auto overflow-x-hidden divide-y divide-gray-200 mb-20"
+        >
+          {nonDODEvidences.map((evidence, index) => {
+            return (
+              <div key={index} className="relative flex items-center p-6">
+                <div className="min-w-0 flex-1 text-sm mr-4">
+                  <EvidenceView evidence={evidence} size={size} />
+                </div>
+                <div className="flex items-center">
+                  <input
+                    type="checkbox"
+                    className="h-4 w-4 cursor-pointer rounded border-gray-300 text-indigo-600 focus:ring-indigo-500"
+                    checked={isSelected(evidence.id)}
+                    onChange={(e) => {
+                      setSelectedEvidences((val) => {
+                        if (val.includes(evidence)) {
+                          return val.filter((v) => v.id !== evidence.id);
+                        } else {
+                          return [...val, evidence];
+                        }
+                      });
+                    }}
+                  />
+                </div>
+              </div>
+            );
+          })}
+          {!Boolean(nonDODEvidences.length) && (
+            <div className="flex items-center justify-center py-5 px-5 h-56">
+              <div className="text-sm text-gray-500">
+                There are no evidences which are not part of defintion of done
+              </div>
+            </div>
+          )}
+        </div>
+        <div
+          className={clsx(
+            "flex rounded-t-lg justify-between bg-gray-100 px-8 pb-4 items-end",
+            "absolute w-full bottom-0 left-0"
+          )}
+        >
+          <div className="flex flex-1">
+            <button
+              type="submit"
+              className={clsx(
+                "inline-flex items-center px-4 py-2 border border-gray-300 shadow-sm font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 sm:text-sm",
```

Comment:
use the standard button classes

Reply:
I think we have used this way in multiple places. should I refactor there as well with btn-primary class